include/frontend/assembler/op-with-info.hpp

```diff
@@ -5,13 +5,14 @@
 #include "ChaiVM/interpreter/autogen/operations.hpp"
 #include "ChaiVM/interpreter/instruction.hpp"
 
-/*
- * @todo #41:90min Rename this class
+/**
+ * Class that encapsulates chai::interpreter::Operation and provides some
+ *  useful methods.
  */
-class OpString {
+class OperationWithInfo {
 public:
-    OpString(chai::interpreter::Operation val) : val_(val) {}
-    explicit OpString(const std::string &strOp) {
+    OperationWithInfo(chai::interpreter::Operation val) : val_(val) {}
+    explicit OperationWithInfo(const std::string &strOp) {
```

Comment:
@c71n93 one of them is explicit, and other is not

Reply:
@levBagryansky as for me constructor from `string` should be explicit. But constructor from `Operation` may be implicit, so I left it implicit.